### PR TITLE
connects to the server to get data

### PR DESCRIPTION
**Pull request by FrancisA2000**

### CODE REVIEW

File: src/components/OrgBubble/index.jsx [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+  }, []);
+
+  const fetchOrgImg = () => {
+    getRequest("/orgImages")
+      .then(res => {
+        res.data.error
+          ? (() => {})()
+          : (() => {
+              setOrgInfo(res.data.orgInfo);
+            })();
```
Comment (Karyum) [inline src/components/OrgBubble/index.jsx]: what is this why ?

File: src/components/OrgBubble/index.jsx [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+    <div className="scrollableDiv_bubble">
+      {orgInfo.map((org, i) => {
+        return (
+          <div key={i} className="org_bubble">
+            <a href={org.fb_url}>
+              <img src={org.img_url}></img>
+            </a>
+          </div>
+        );
+      })}
```
Comment (Karyum) [inline src/components/OrgBubble/index.jsx]:
```suggestion
      {orgInfo.map((org, i) => (
          <div key={i} className="org_bubble">
            <a href={org.fb_url}>
              <img src={org.img_url}></img>
            </a>
          </div>
        )
      )}
```

File: src/components/OrgStatistic/index.jsx [hunk 3]
```diff
@@ -1,85 +1,81 @@
...
-import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
-import Typography from '@material-ui/core/Typography';
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import Typography from "@material-ui/core/Typography";
+import Cookie from "js-cookie";
+import { getRequest } from "../../utils/axios.js";
```
Comment (Karyum) [inline src/components/OrgStatistic/index.jsx:8]: again your prettier, you really need to fix it, either you or @92yo

File: src/components/OrgStatistic/index.jsx [hunk 4]
```diff
@@ -1,85 +1,81 @@
...
+  const fetchOrgStatistic = () => {
+    getRequest("/UserStatistics", token)
+      .then(res => {
+        res.data.error
+          ? (() => {
+              window.location = "/";
+            })()
+          : (() => {
+              setOrgStatistic(res.data.donations);
+            })();
```
Comment (Karyum) [inline src/components/OrgStatistic/index.jsx]: again why ?

**Review by Karyum — CHANGES_REQUESTED** [review]

File: src/components/OrgStatistic/index.jsx (hunk 4 above)
Comment (FrancisA2000) [reply src/components/OrgStatistic/index.jsx]: error handeling

**Review by Karyum — APPROVED** [review]

**Pull request merged**